CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 BREAKING CHANGES:
 
+Don't prepend the `.my.cnf` file with the `HOME` environment.
```

Comment:
As a user, this description is not helpful to me. Does this mean I should not use `$HOME` when specifying the flag? What happens differently when I specify some file name?

I _think_ what actually changes from a user perspective is

```suggestion
The default client configuration file is now `.my.cnf` in the process working directory. Use `--config.my-cnf="$HOME/.my.cnf"` to retain the previous default.
```